### ISSUE TITLE
Timeline

### ISSUE THREAD

**Issue by dlebedynskyi** [react-toolbox/react-toolbox#593]
hey guys, 
on our project we had to create a timeline component. I thought it might be handy for this project .
It is nothing special just combination of basic stuff, some animation. Here is gist with source
https://gist.github.com/dlebedynskyi/64a28f366cdd41d6ab01e957d4a0d755 

Sample of usage 
```
<Timeline>
<TimelineItem date="2016-06-24"><Card> .... </Card></TimlineItem>
<TimelineItem date="2016-06-24"  details="Some text" inverse><Card> .... </Card></TimlineItem>
</Timeline>
```

I can work on pull if you guys need this

**Comment by javivelasco** [comment]
Is this one documented in the Material spec?? In my opinion the best would be to keep components from the spec. If it's in the spec your PR would be very welcome. In other case I'd be happy to add links with "complementary" components that fits with RT project structure. What do you think?

**Comment by dlebedynskyi** [comment]
@javivelasco  I don't think this is in material design spec. If you want you can add this to complementary i guess. I'll think on creating separate repo this one.

**Comment by gedw99** [comment]
@javivelasco @dlebedynskyi  

I would like to know what to do also. 

I have also been building various react components on this react-toolbox. For example:
- Calendar with Scheduler
- Document Editor
- Document Data mapper

Its great that your strict on making sure things follow the MD spec for the core controls. Me and others are using those building block to make bigger components, and and knowing that the core controls are high quality helps allot.

But it would be great to have a place where its easy to find these higher level components, because i and others often need them. They also act as examples for others and grow the knowledge. And of course it saves heaps of time

**Comment by javivelasco** [comment]
I think it's a *great* idea to create a separate repository for complementary components. We can create like a `react-toolbox-complements` repository with the same structure. I would not mix everything here although it would be more handy.

I would be awesome to get opinions on this. 👍 for a new repository 👎 to integrate complementary components (out of the spec) in here please! 🚀

**Comment by dlebedynskyi** [comment]
@javivelasco  sounds like a good idea. How about you create a repo and then we send pull into it?

**Comment by javivelasco** [comment]
Closed in favor of #626
